recipe/migrations/openturns1201.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+migrator_ts: 1665598497.9972937
+openturns:
+- '1.201'
```

To confirm, this should be...?

```suggestion
- '1.20'
```